### DISCUSSION TITLE
Using Imgr as a way to save images

### DISCUSSION THREAD

**Discussion by kaiakrystal** [flameshot-org/flameshot#3176] [Q&A] [answered]
Is there a way to choose where you want to download images to share with others. Imgr is going to be going through and deleting pretty much anything that depicts NSFW content, specifically sexual related images, as well as block any such image downloads from happening.

**Comment by mmahmoudian** [comment, marked as answer]
I think you meant "upload" and not "download". You cannot in the current version upload automatically anywhere else from Flameshot. You have to do it manually, for example through your browser.
